### PR TITLE
#168781652: like and unlike accommodation

### PR DESCRIPTION
**Pull request by alainmateso**
#### What does this PR do?
Like and unlike accommodations

#### Description of Task to be completed?
- users can like accommodations
- users can unlike accommodations
- users can see the total number of likes and dislikes of an accommodation

#### How should this be manually tested?

- Clone this branch 
- run `npm i`
- run `npm run dev`
- navigate to` http://localhost:<:port>/accommodations/:slug`
- press like or dislike button

#### Screenshots (if appropriate)
Screen Shot 2019-12-31 at 11 54 51
Screen Shot 2019-12-31 at 11 55 21

#### Questions:
N/A

### CODE REVIEW

**Review by ngireric123 — COMMENTED** [review]
@alainmateso  good job. Kindly increase your test coverage to 90%

**Review by sabin18 — CHANGES_REQUESTED** [review]
Good job @alainmateso. However, make likes more visible on view  all accommodations.

**Review by ngireric123 — APPROVED** [review]
LGTM

**Review by sabin18 — APPROVED** [review]

File: src/components/pages/SingleAccommodation.js [hunk 1]
```diff
@@ -25,13 +30,15 @@ export class SingleAccommodation extends React.Component {
       isLoading: false,
       userId: null,
     };
+    this.handleLike = this.handleLike.bind(this);
```
Comment (nakiwuge) [inline src/components/pages/SingleAccommodation.js]: Do not bind be consistent

File: src/styles/pages/allAccommodation.scss [hunk 2]
```diff
@@ -80,3 +80,7 @@
   padding-left: 20px;
 }
 
+.like{
+  font-weight: bolder;
+  color: $primary;
+}
```
Comment (nakiwuge) [inline src/styles/pages/allAccommodation.scss:99]: Add a line at the end of the file

File: src/tests/componentTests/likeUnlikeTest.spec.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+      </Router>
+    </Provider>
+  );
+}
+describe('Like and dislike Accommodation Test Suite', () => {
+    it('Should handle like', () => {
+        const spying = sinon.spy(axios, 'post');
+        axios.patch.mockResolvedValue({message: 'Liked successfully'});
+        let wrapper = makeWrapper();
+        console.log('wrapper', wrapper.html())
```
Comment (nakiwuge) [inline src/tests/componentTests/likeUnlikeTest.spec.js]: Remove console.log

File: src/tests/actionsTests/likeUnlikeActionTest.spec.js [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            const request = moxios.requests.mostRecent();
+            request.respondWith(likeFailed);
+            await cleanUp();
+        });
+        await store.dispatch(likeUnlikeAccommodation('slug', 'like'));
+        const calledAction = store.getActions();
+
+        expect(calledAction[0].type).toEqual(LIKE_ACCOMMODATION_ERROR)
+    });
+});
```
Comment (nakiwuge) [inline src/tests/actionsTests/likeUnlikeActionTest.spec.js]: Add a line at the end of the file

File: src/tests/componentTests/likeUnlikeTest.spec.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+        expect(spying.calledOnce);
+    });
+
+});
```
Comment (nakiwuge) [inline src/tests/componentTests/likeUnlikeTest.spec.js]: Add a line at the end of the file

File: src/tests/componentTests/likeUnlikeTest.spec.js [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+}
+describe('Like and dislike Accommodation Test Suite', () => {
+    it('Should handle like', () => {
+        const spying = sinon.spy(axios, 'post');
+        axios.patch.mockResolvedValue({message: 'Liked successfully'});
+        let wrapper = makeWrapper();
+        console.log('wrapper', wrapper.html())
+        expect(spying.calledOnce);
+    });
+
```
Comment (nakiwuge) [inline src/tests/componentTests/likeUnlikeTest.spec.js]: Remove this line

**Review by nakiwuge — CHANGES_REQUESTED** [review]
@alainmateso this is good work however kindly add thumbs up and thumbs down icons on each accommodation on the `get all accommodations` page and give each a different color.

I have left other comments on the pr.

**Review by nakiwuge — APPROVED** [review]

**Pull request merged**